.releaserc

```diff
@@ -0,0 +1,30 @@
+{
+	"branches": ["version-13"],
+	"plugins": [
+		"@semantic-release/commit-analyzer", {
+			"preset": "angular",
+			"releaseRules": [
+				{"breaking": true, "release": false}
+			]
+		},
+		"@semantic-release/release-notes-generator",
+		[
+			"@semantic-release/exec", {
+				"prepareCmd": 'sed -ir "s/[0-9]*\.[0-9]*\.[0-9]*/${nextRelease.version}/" frappe/__init__.py'
+			}
+		],
+		[
+			"@semantic-release/git", {
+				"assets": ["frappe/__init__.py"],
+				"message": "chore(release): Bumped to Version ${nextRelease.version}\n\n${nextRelease.notes}"
+			}
+		],
+		[
+			"@semantic-release/github", {
+				"assets": [
+					{"path": "dist/*"},
```

Comment:
This doesn't seem to be requried. Assets are separately published by another workflow to existing releases. 

Just a GitHub release should be enough.